### ISSUE TITLE
[Topic] Headings in the content

### ISSUE THREAD

**Issue by rianrietveld** [wpaccessibility/wp-a11y-docs#106]
URL: https://wpaccessibility.org/docs/topics/content/headings/
Code: https://github.com/wpaccessibility/wp-a11y-docs/blob/main/docs/topics/content/headings.md

Please read [How to contribute](https://wpaccessibility.org/docs/contribute/) before you start.

## Tasks

- this content should be reviewed
- the links should be checked and updated. More recent links could be added.
